test/README.md

```diff
@@ -41,6 +41,74 @@ The following test modules are defined for corresponding `torchaudio` module/fun
 - [assets/kaldi](./assets/kaldi): Contains Kaldi format matrix files used in [./test_compliance_kaldi.py](./test_compliance_kaldi.py).
 - [compliance](./compliance): Scripts used to generate above Kaldi matrix files.
 
+### Waveforms for Testing Purposes
+
+When testing transforms we often need waveforms of specific type (ex: pure tone, noise, or voice), with specific bitrate (ex. 8 or 16 kHz) and number of channels (ex. mono, stereo). Below are some tips on how to construct waveforms and guidance around existing audio files.
+
+#### Load a Waveform from a File
+
+```python
+filepath = common_utils.get_asset_path('filename.wav')
+waveform, sample_rate = scipy.io.wavfile.read(filepath)
+```
+
+*Note: Should you choose to contribute an audio file, please leave a comment in the issue or pull request, mentioning content source and licensing information. WAV files are preferred. Other formats should be used only when there is no alternative. (i.e. dataset implementation comes with hardcoded non-wav extension).*
+
+#### Pure Tone
+
+Code:
+
+```python
+ waveform = common_utils.get_sinusoid(
+     frequency=300,
+     sample_rate=16000,
+     duration=1,  # seconds
+    n_channels=1,
+    dtype="float32",
+    device="cpu",
+)
+```
+
+Files:
+
+* `sinewave.wav`
+* `100Hz_44100Hz_16bit_05sec.wav`
+* `440Hz_44100Hz_16bit_05sec.wav`
+
+#### Noise
+
+Code:
+
+```python
+tensor = common_utils.get_whitenoise()
+```
+
+Files:
+
+* `whitenoise.wav`
+* `whitenoise.mp3`
+* `whitenoise_1min.mp3`
+* `steam-train-whistle-daniel_simon.wav`
+* `steam-train-whistle-daniel_simon.mp3`
```

Comment:
I suggest to remove `mp3` version when there is an equivalent `wav`.
I checked the test but there is no need to use the current `mp3` files except I/O test, and that will make it easier for us to get rid of these `mp3` files.